include/ebpf_nethooks.h

```diff
@@ -175,6 +176,23 @@ EBPF_HELPER(uint64_t, bpf_sock_addr_get_current_pid_tgid, (bpf_sock_addr_t * ctx
     ((bpf_sock_addr_get_current_pid_tgid_t)BPF_FUNC_sock_addr_get_current_pid_tgid)
 #endif
 
+/**
+ * @brief Set a context for consumption by a proxy application (sock_addr specific only).
+ * This function is not supported for the recv_accept hooks.
+ *
+ * @param[in] ctx Pointer to bpf_sock_addr_t context.
+ * @param[in] data Pointer to data to store.
+ * @param[in] data_size The size of the data to store.
+ *
+ * @retval 0 The operation was successful.
+ * @retval <0 A failure occurred.
```

Comment:
[Nit]: Add a whitespace after <,  i.e., < 0

Reply:
This looks consistent with all existing usages in our codebase. I will leave this as is.